Source/Fields/LMCCylindricalCavity.hh

```diff
@@ -48,6 +48,7 @@ namespace locust
             virtual double Integrate(int l, int m, int n, bool teMode, bool eField);
             virtual std::vector<double> GetDopplerFrequency(int l, int m, int n, std::vector<double> tKassParticleXP);
             virtual std::vector<double> GetNormalizedModeField(int l, int m, int n, std::vector<double> tKassParticleXP, bool includeOtherPols);
+	    virtual std::vector<double> GetNormalizedModeField(int l, int m, int n, std::vector<double> tKassParticleXP, bool includeOtherPols, bool teMode);
```

Comment:
Can these two GetNormalizedModeField() functions be consolidated into one function, or do we need both of them?